README.md

```diff
@@ -1,11 +1,14 @@
 # postfix-docker 📮🐳 #
 
 [![Build Status](https://travis-ci.com/cisagov/postfix-docker.svg?branch=develop)](https://travis-ci.com/cisagov/postfix-docker)
```

Comment:
Do we still need this?

Reply:
Yup.  That one snuck through.  Good catch.  Removed in https://github.com/cisagov/postfix-docker/pull/5/commits/a4867e45c0d3daf1319b7f406eb1910e5c3dcf78.